content/courses/advanced-ada/parts/data_types/types_representation.rst

```diff
@@ -2775,6 +2775,238 @@ these array declarations are equivalent:
     end Shared_Var_Types;
 
 
+:ada:`Full_Access_Only`
+~~~~~~~~~~~~~~~~~~~~~~~
+
+A full access object is an object that requires that read or write operations
+on this object be performed by reading or writing all bits of the object (i.e.
```

Comment:
"this object be performed" is a little awkward. Maybe "this object are performed"  instead.

Reply:
Thanks for the correction!